app/models/workers_scaling/algorithms/resources_usage_minimization.rb

```diff
@@ -68,53 +67,53 @@ def time_constraint_check(predicted, left)
     ##
     # Increases computational power to match required system throughput
     # Does nothing if there are starting Workers already
-    # Uses known infrastructures with highest throughput if limits are not reached
-    # If limits in all known infrastructures are reached, uses random unknown infrastructure
+    # Uses known configurations with highest throughput if limits are not reached
+    # If limits in all known configurations are reached, uses random unknown configuration
     def increase_computational_power
-      LOGGER.debug 'Need to increase computational power'
+      log(:debug, 'Need to increase computational power')
```

Comment:
To prawda - na produkcji logi nie cierpia. Cierpia natomiast wszedzie indziej + jesli logowanie nic nie wnosi to jest szkodliwe bo zwieksza ilosc kodu a zatem zmniejsza czytelnosc metody.